modules/nodes-containers-port-forwarding-using.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-containers-port-forwarding.adoc
+
+[id='nodes-containers-port-forwarding-using_{context}']
+= Using port forwarding
+
+You can use the CLI to port-forward one or more local ports to a pod.
+
+.Procedure
+
+* Use the following command to listen on ports `5000` and `6000` locally and forward data to and from ports `5000` and `6000` in the pod:
```

Comment:
I think you need to have one explicit command to forward the port and then some examples. Do we have a larger flow that this fits into?